lib/tasks/batch_connect.rake

```diff
@@ -0,0 +1,31 @@
+namespace :batch_connect do
+  desc "Generate new batch connect session"
+  task new_session: :environment do
+    # Read in user settings
+    app = ENV["BC_APP_TOKEN"] || abort("Missing environment variable BC_APP_TOKEN")
+    fmt = ENV["BC_RENDER_FORMAT"]
+    ctx = if !$stdin.tty?
```

Comment:
> I guess the answer depends on what the reason was for adding this rake task in the first place... 

I thought the reason was so other apps or utilities could launch iHPC sessions on behalf of the user. So reading from stdin may not always be the easiest option depending on the language chosen by the developer.